flake.nix

```diff
@@ -0,0 +1,47 @@
+{
+  description = "A Nix-flake-based development environment for Cursorless";
+
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
+
+  outputs = { self, nixpkgs }:
+    let
+      overlays = [
+        (final: prev: rec {
+          nodejs = prev.nodejs-18_x;
+          pnpm = prev.nodePackages.pnpm;
+          yarn = (prev.yarn.override { inherit nodejs; });
+        })
+      ];
+      supportedSystems =
+        [ "x86_64-linux" "aarch64-linux" "x86_64-darwin" "aarch64-darwin" ];
+      forEachSupportedSystem = f:
+        nixpkgs.lib.genAttrs supportedSystems (system:
+          f {
+            pkgs = import nixpkgs {
+              inherit overlays system;
+              config.allowUnfree = true;
+            };
+          });
+    in {
+      devShells = forEachSupportedSystem ({ pkgs }: {
+        default = pkgs.mkShell {
+          packages = with pkgs; [
+            node2nix
```

Comment:
Does node2nix support pnpm? I thought it only supports npm.

But also, Cursorless currently doesn't get published as a library or CLI tool, so I'm not sure *2nix is useful in our case.

Reply:
No idea tbh. I will remove and make sure everything still works.

Reply:
Looks like there is a pnpm2nix package instead, so you are right it wouldn't have worked anyway. But removed, since it's not needed.